Java/corecrypto/src/main/java/com/breadwallet/corecrypto/Network.java

```diff
@@ -264,11 +239,60 @@ public boolean hasCurrency(com.breadwallet.crypto.Currency currency) {
         return unitsFor(currency).transform(input -> input.contains(unit));
     }
 
+    public void addCurrency(com.breadwallet.crypto.Currency currency,
+                            com.breadwallet.crypto.Unit baseUnit,
+                            com.breadwallet.crypto.Unit defaultUnit) {
+    }
+
+    public void addUnitFor(com.breadwallet.crypto.Currency currency, com.breadwallet.crypto.Unit unit) {
+    }
+
```

Comment:
Are these TODOs?

Reply:
Over looked; still need to run Unit tests and Java Demo App...  Doing now.